### PR TITLE
New pack CTF Cloud usecase

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+                "sort": [
+                    {
+                        "field": "id",
+                        "fieldType": "",
+                        "asc": false
+                    }
+                ],
+                "Cache": null,
+                "name": "Finished CTF Incidents",
+                "prevName": "List of all CTF incidents",
```
Comment (julieschwartz18) [inline Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json:223]: Maybe should be capitalized?
List of All CTF Incidents

File: Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+                    "all"
+                ],
+                "definitionId": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "commitMessage": "",
+                "shouldCommit": false,
+                "Cache": null,
+                "name": "Fastest Top 10 users finished Advanced CTF",
+                "prevName": "Top 10 users finished",
```
Comment (julieschwartz18) [inline Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json:413]: Capitalized?
Top Ten Users to Finish

File: Packs/CTFCloud/Scripts/APIKeyRevokeCTF/APIKeyRevokeCTF.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+args:
+- name: accesskey
+  required: true
+comment: CTF script that initiates a revoke for all user access keys.
```
Comment (ssokolovich) [inline Packs/CTFCloud/Scripts/APIKeyRevokeCTF/APIKeyRevokeCTF.yml:4]:
```suggestion
comment: CTF script that initiates a revoke an access key.
```

Comment (julieschwartz18) [reply Packs/CTFCloud/Scripts/APIKeyRevokeCTF/APIKeyRevokeCTF.yml]: I suggest:
CTF script that initiates revoking an access key.

File: Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json (hunk 1 above)
Comment (ssokolovich) [reply Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json:223]: No need, that is the previous name -> name is the right field to look at it.

File: Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json (hunk 2 above)
Comment (ssokolovich) [reply Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json:413]: No need, that is the previous name -> name is the right field to look at it.

**Pull request closed without merging**